Animate/zoomIn.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title> zoomIn | Animate</title>
+    <style>
+        #zoomIn {
+            position: relative;
+            color: grey;
+            animation: zoomIn 2s;
+            -webkit-animation-iteration-count: infinite;
+            animation-iteration-count: infinite;
+        }
+       
+        @keyframes zoomIn {
+            0% {
+                transform: scale(0.2);
+                opacity: 0;
+            }
+            60% {
+                transform: scale(1.4);
+                opacity: 1;
+            }
+            100% {
+                transform: scale(1);
+            }
+        }
+    </style>
+</head>
+
+<body>
+    <div id="zoomIn">The quick brown fox jumps over the lazy dog</div>
```

Comment:
There should be nothing between div. For content, please use `:before` and `:after` pseudo elements.